### PR TITLE
Remove old and add new framework target.

### PR DESCRIPTION
**Pull request by paulyoung**
This follows on from #449.

We were unable to build our app once adding the new AsyncDisplayKit framework in various different ways.

After cloning this repo I was still unable to build the framework target in the project itself.

I removed the old target and added a new one. This builds fine stand-alone and as part of our app.

Please review :pray:

### CODE REVIEW

**Comment by appleguy** [comment]
Hmm, this conflicts with another PR I just merged.  Sorry I didn't see this one first.  Could you check that PR, and determine if this one is still necessary?  If it's necessary, update it so it merges cleanly and I will merge it for you promptly.  Thank you @paulyoung !

**Comment by paulyoung** [comment]
@appleguy I think you're referring to #507 which I hadn't seen.

I think this one is now only necessary if you'd prefer the consistent naming that comes with adding a brand new target.

I think there was some discussion in #449 which led to the target and/or product to being renamed part way through.

**Comment by appleguy** [comment]
I would love the consistency!  If you can get this into a state where it merges cleanly, I'll take the diff.  Alternatively if you won't have a chance, we can close and revisit another time.  Thank you :).

**Comment by paulyoung** [comment]
@appleguy done. Please review!

File: AsyncDisplayKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1587,9 +1590,9 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				GCC_NO_COMMON_BLOCKS = YES;
-				INFOPLIST_FILE = AsyncDisplayKit/Info.plist;
+				INFOPLIST_FILE = "AsyncDisplayKit-iOS/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.3;
```
Comment (paulyoung) [inline AsyncDisplayKit.xcodeproj/project.pbxproj]: I noticed that there are 2 places where the framework target is now set to 8.3

This was the default. I can change it back if necessary.

**Comment by paulyoung** [comment]
It seems like Travis may be having issues but I'm unable to restart the build to confirm.

**Comment by appleguy** [comment]
Restarting build...

**Comment by nickvelloff** [comment]
@appleguy how'd the build go?

**Comment by appleguy** [comment]
Thanks for the ping — today has been heads-down on ASDataController (watch for another diff coming tonight or tomorrow; the good stuff!)

Glad to see the build succeeded, so you are in.

**Comment by paulyoung** [comment]
Thanks @appleguy!

**Comment by nickvelloff** [comment]
Yup thanks @appleguy!

**Pull request merged**